addons/analytic/models/analytic_account.py

```diff
@@ -93,6 +136,41 @@ class AccountAnalyticLine(models.Model):
     def _default_user(self):
         return self.env.context.get('user_id', self.env.user.id)
 
+
+
+    def _compute_amount_user_currency(self):
+        """ Compute the amount into the user's currency
+        """
+        context = dict(self._context or {})
+        user_currency_id = self.env.user.company_id.currency_id
+        ctx = context.copy()
+        for line in self:
+            ctx['date'] = line.date
+            company_currency_id = line.company_id.currency_id
+            amount = line.amount
+            if user_currency_id == company_currency_id:
+                line.amount_user_currency = amount
+            else:
+                line.amount_user_currency = company_currency_id.with_context(ctx).compute(amount, user_currency_id)
+
+    @api.model
+    def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=False, lazy=True):
+        res = super(AccountAnalyticLine, self).read_group(domain, fields, groupby, offset=offset, limit=limit, orderby=orderby, lazy=lazy)
+        # We compute the amount that have to be displayed into the user's currency
+        # We must call '_compute_amount_user_currency' manually, because of the pivot view not doing it.
+        if 'amount_user_currency' in fields:
+            self._compute_amount_user_currency()
+            for line in res:
+                __domain = list('__domain' in line and line['__domain'] or [])
```

Comment:
I can't call it domain since there's already a parameter named domain, the list is pointless ?

Should I do ```python
__domain = '__domain' in line and line['__domain'] or []
```